Rakefile

```diff
@@ -3,6 +3,20 @@ require 'rake/clean'
 
 Dir['tasks/**/*.rake'].each { |t| load t }
 
-task local_test_run: [:test, :rubocop, 'test:quality']
-task ci: [:test, :rubocop, 'test:quality', :ataru]
+defaults = ['test:spec',
+            # Should run before test:features so we have the the most up-to-date default configuration
```

Comment:
This is no longer needed since the `defaults.reek.yml` configuration file is only for documentation now.

Reply:
Good point, removed the comment.